packages/pluginsdk-peerdeps/sync-versions-from-deck.js

```diff
@@ -16,8 +16,9 @@ const yargs = require('yargs')
     description: 'The destination package.json',
     default: './package.json',
   })
-  .option('no-dev', { type: 'boolean', description: 'do not sync devDependencies', default: false })
-  .option('no-peer', { type: 'boolean', description: 'do not sync peerDependencies', default: false });
+  .option('spinnaker', { type: 'boolean', description: 'include @spinnaker/* dependencies', default: true })
```

Comment:
flipped config from e.g. `no-dev` to `dev` and flipped defaults.  This can be used from the command line like:
`--no-dev`
`--dev false`
`--dev=false`